### DISCUSSION TITLE
Does anyone have comparison over TTS models available in Coqui?

### DISCUSSION THREAD

**Discussion by JaejinCho** [coqui-ai/TTS#2268] [General Q&A]
Hello and happy new year!

Thank you for opening this great repo to the public. I was wondering if there is some comparison of available TTS models here under the same condition w.r.t synthesized speech quality (either based on MOS or generated samples) and RTF. 

I have been looking for them and what I have found so far are below:
- https://github.com/coqui-ai/TTS/discussions/522
- chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/https://arxiv.org/pdf/2104.05557.pdf
- chrome-extension://efaidnbmnnnibpcajpcglclefindmkaj/https://proceedings.mlr.press/v162/casanova22a/casanova22a.pdf

It would be great if someone could share other references regarding the comparison other than what I've found so far.

If I may share, my final goal is to start working from a few TTS models that are better than others, based on the above comparison instead of trying all models.

Sincerely,
Jaejin Cho

**Comment by erogol** [comment]
There is a thread in the discussion. You can search for it ...
